checkout-core/src/main/java/com/adyen/checkout/core/api/Environment.java

```diff
@@ -47,6 +48,7 @@ public Environment createFromParcel(@NonNull Parcel in) {
         try {
             TEST = new Environment(new URL("https://checkoutshopper-test.adyen.com/checkoutshopper/"));
             EUROPE = new Environment(new URL("https://checkoutshopper-live.adyen.com/checkoutshopper/"));
+            LIVE = EUROPE;
```

Comment:
Can you put the changes in only one commit? Also please put `LIVE = EUROPE` below the instantiation lines, i think that's better for readability. Other than that looks good! 👍

Reply:
done 👍